test/unittests/api/v0.0.1/Candidates_Test.php

```diff
@@ -97,17 +96,52 @@ function testPostCandidateValid() {
                     ]
                 ]
                 );
+            return $API;
         } catch(\Loris\API\SafeExitException $e) {
             $API = $e->Object;
+            return $API;
         }
+    }
+    function testPostCandidateValid() {
```

Comment:
And, in the comment below, I say that our unit tests are structured in a way that just breaks the test when the feature actually works.

Modifying random generation was not part of this pull request but that's the problem after all of my debugging above. As for why random generation fails, I have no idea. I'm not a unit test dude and I didn't set it all up.

![image](https://user-images.githubusercontent.com/5655961/26890256-88642008-4be3-11e7-8dae-a793e575ebd0.png)